projects/storefrontapp-e2e-cypress/cypress/integration/regression/product_configurator/product-configurator-vc-conflict-dialog-e2e-spec-flaky.ts

```diff
@@ -0,0 +1,84 @@
+import * as conflictDialog from '../../../helpers/product-configurator-conflict-dialog';
+import * as configuration from '../../../helpers/product-configurator';
+import * as configurationVc from '../../../helpers/product-configurator-vc';
+import * as configurationOvVc from '../../../helpers/product-configurator-overview-vc';
+import { clickAllowAllFromBanner } from '../../../helpers/anonymous-consents';
+
+const electronicsShop = 'electronics-spa';
+const testProduct = 'CONF_CAMERA_SL';
+
+const RB = 'radioGroup';
+
+context('Product Configuration', () => {
+  beforeEach(() => {
+    cy.visit('/');
+    configurationVc.registerConfigurationRoute();
```

Comment:
@Uli-Tiger, what about `registerGetConfigurationRoute`?

Reply:
As i did not introduce this method and as it used within most cypress tests, i would suggest not to rename it, as i it would enlarge the PR.